src/C0_data_processing.py

```diff
@@ -203,13 +203,22 @@ def energyConversion(dict_values, group):
     """
     #
     for asset in dict_values[group]:
+        print("asset", asset)
         define_missing_cost_data(dict_values, dict_values[group][asset])
         evaluate_lifetime_costs(
             dict_values["simulation_settings"],
             dict_values["economic_data"],
             dict_values[group][asset],
         )
 
```

Comment:
```suggestion
    # To simulate the standby power of electrolyzers, a standby consumption sink is added. Minimal variable costs are added because...??
```

Why does this standby consumption sink have a variable cost connected to the flow?